listings/applications/components/src/countable.cairo

```diff
@@ -24,3 +27,68 @@ pub mod countable_component {
         }
     }
 }
+// ANCHOR_END: countable_component
+
+// ANCHOR: MockContract
+#[starknet::contract]
+mod MockContract {
+    use super::countable_component;
+
+    component!(path: countable_component, storage: countable, event: countableEvent);
+
+    #[storage]
+    struct Storage {
+        #[substorage(v0)]
+        countable: countable_component::Storage,
+    }
+
+    #[event]
+    #[derive(Drop, starknet::Event)]
+    enum Event {
+        countableEvent: countable_component::Event
+    }
+
+    #[abi(embed_v0)]
+    impl CountableImpl = countable_component::Countable<ContractState>;
+}
+// ANCHOR_END: MockContract
+
+#[cfg(test)]
+mod test {
+    use super::MockContract;
+    use components::countable::{ICountableDispatcher, ICountableDispatcherTrait};
+    use starknet::syscalls::deploy_syscall;
+    use starknet::SyscallResultTrait;
+
+    fn deploy_countable() -> ICountableDispatcher {
+        let (address, _) = deploy_syscall(
+            MockContract::TEST_CLASS_HASH.try_into().unwrap(), 0, array![].span(), false
+        )
+            .unwrap_syscall();
+        ICountableDispatcher { contract_address: address }
+    }
+
+    #[test]
+    fn test_constructor() {
+        let counter = deploy_countable();
+        assert_eq!(counter.get(), 0);
+    }
+
+    #[test]
+    fn test_increment() {
+        let counter = deploy_countable();
+        counter.increment();
+        assert_eq!(counter.get(), 1);
+    }
+
+    #[test]
+    fn test_multiply_increment() {
```

Comment:
```suggestion
    fn test_multiple_increments() {
```
nit: the name wrongly implies the counter is being "multiplied", changing it to "multiple_increments" states the test is incrementing multiple times.